test/RequestStreamTest.cpp

```diff
@@ -72,11 +98,12 @@ class TestHandlerAsync : public rsocket::RSocketResponder {
               return Payload(s, "metadata");
             })
             ->subscribe(subscriber);
-      }).detach();
+      })
+          .detach();
```

Comment:
is this clang-format doing something weird?

Reply:
must've been, looks weird to me, I'll try to put it back